test/fuzz/test_runner.py

```diff
@@ -56,6 +56,14 @@ def main():
         '--m_dir',
         help='Merge inputs from this directory into the seed_dir. Needs /target subdirectory.',
     )
+    parser.add_argument(
+        '-g',
+        '--generate',
+        action='store_true',
+        help='Create new corpus seeds by running the given targets by batches '
```

Comment:
```suggestion
        help='Create new corpus seeds (or extend the existing ones) by running the given targets by batches '
```

This will also take into account existing seeds, right?

Reply:
Yep